acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/comparison/CompareSignMessageAcceptanceTest.java

```diff
@@ -41,33 +40,34 @@ void initSigner() {
 
   @Test
   void compareRandomMessageSignaturesBetweenLotusAndEthSigner() {
+    final int signatureCount =
```

Comment:
why does this need to be a sys property, is this going to be overridden in circleCI?

Reply:
🤷 Having it as an arbitrary constant seemed odd - this allows a degree of flexibility - happy to revert ...